flink-connectors/flink-connector-base/src/test/java/org/apache/flink/connector/base/source/reader/synchronization/FutureCompletingBlockingQueueTest.java

```diff
@@ -0,0 +1,22 @@
+package org.apache.flink.connector.base.source.reader.synchronization;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```

Comment:
Can you add a license header to the file? Also, we usually do not use `*` in the imports.

Reply:
@becketqin Thanks for your detail guidance . I will fix it